src/system-health/health_checker/sysmonitor.py

```diff
@@ -109,17 +128,72 @@ def task_notify(self, msg):
             return
         self.task_queue.put(msg)
 
+
+# Subprocess to monitor system ready timeout. If timeout will be exceeded,
+# send a message to queue and exit
+class MonitorTimeout(ProcessTaskBase):
```

Comment:
> This thread needs to be spawned only when feature is enabled.

@dgsudharsan not only, it should be spawned also when feature is disabled, cause when it is disabled it still wait for one state - swss's `PortInitDone`